docs/AboutUs.adoc

```diff
@@ -3,7 +3,7 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
+AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[Team T09-B3] team. +
```

Comment:
Do we need to leave this be as a credit to the original creators? I don't see any mentions of required changes to this part in the module website.

Reply:
No. Infact we need to replace it with our teams description

Reply:
Ok, then that means the hyperlink toe se-edu should be removed as well as the dummy line below?

Reply:
No. We have to change as per our project.